### PR TITLE
[zh] Update admission-controllers.md

### PR DESCRIPTION
**Pull request by sniperking1234**
Fix statement order problem and translate.

### CODE REVIEW

**Comment by sniperking1234** [comment]
signed

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 1]
```diff
@@ -274,8 +274,8 @@ This admission controller ignores any `PersistentVolumeClaim` updates; it acts o
 -->
 当未配置默认存储类时，此准入控制器不执行任何操作。如果将多个存储类标记为默认存储类，
 它将拒绝任何创建 `PersistentVolumeClaim` 的操作，并显示错误。
-此时准入控制器会忽略任何 `PersistentVolumeClaim` 更新操作，仅响应创建操作。
 要修复此错误，管理员必须重新访问其 `StorageClass` 对象，并仅将其中一个标记为默认。
+此准入控制器会忽略任何 `PersistentVolumeClaim` 更新操作，仅响应创建操作。
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:278]: any 在这里译为”所有“更合适

Comment (sniperking1234) [reply content/zh/docs/reference/access-authn-authz/admission-controllers.md]: Done.

**Comment by tengqm** [comment]
Looks like we have some other changes to be synced for this page.

```
$ ./scripts/lsync.sh content/zh/docs/reference/access-authn-authz/admission-controllers.md
diff --git a/content/en/docs/reference/access-authn-authz/admission-controllers.md b/content/en/docs/reference/access-authn-authz/admission-controllers.md
index 9bd082dc1..e57813ec1 100644
--- a/content/en/docs/reference/access-authn-authz/admission-controllers.md
+++ b/content/en/docs/reference/access-authn-authz/admission-controllers.md
@@ -725,10 +725,15 @@ See the [resourceQuota design doc](https://git.k8s.io/community/contributors/des
 
 ### RuntimeClass {#runtimeclass}
 
-{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.20" state="stable" >}}
 
-For [RuntimeClass](/docs/concepts/containers/runtime-class/) definitions which describe an overhead associated with running a pod,
-this admission controller will set the pod.Spec.Overhead field accordingly.
+If you enable the `PodOverhead` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/), and define a RuntimeClass with [Pod overhead](/docs/concepts/scheduling-eviction/pod-overhead/) configured, this admission controller checks incoming
+Pods. When enabled, this admission controller rejects any Pod create requests that have the overhead already set.
+For Pods that have a  RuntimeClass is configured and selected in their `.spec`, this admission controller sets `.spec.overhead` in the Pod based on the value defined in the corresponding RuntimeClass.
+
+{{< note >}}
+The `.spec.overhead` field for Pod and the `.overhead` field for RuntimeClass are both in beta. If you do not enable the `PodOverhead` feature gate, all Pods are treated as if `.spec.overhead` is unset.
+{{< /note >}}
 
 See also [Pod Overhead](/docs/concepts/scheduling-eviction/pod-overhead/)
```

**Comment by sniperking1234** [comment]
@tengqm Synced.

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 2]
```diff
@@ -1224,20 +1224,35 @@ See the [resourceQuota design doc](https://git.k8s.io/community/contributors/des
...
 
-{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
++{{< feature-state for_k8s_version="v1.20" state="stable" >}}
 
-[RuntimeClass](/zh/docs/concepts/containers/runtime-class/) 定义描述了与运行 Pod
-相关的开销。此准入控制器将相应地设置 `pod.spec.overhead` 字段。
+如果你开启 `PodOverhead`
+[特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/),
+并且通过[Pod 开销](/zh/docs/concepts/scheduling-eviction/pod-overhead/)
+配置来定义一个容器运行时类， 这个准入控制器会检查新的 Pod。
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:1247]: RuntimeClass 连在一起书写时，不要翻译。

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 3]
```diff
@@ -1224,20 +1224,35 @@ See the [resourceQuota design doc](https://git.k8s.io/community/contributors/des
...
+当启用的时候，这个准入控制器会拒绝任何 overhead 字段已经设置的 Pod。
+对于具有容器运行时类并在入选其 `.spec` 的Pod，
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:1249]:
```suggestion
对于配置了 RuntimeClass 并在其 `.spec` 中选定 RuntimeClass 的Pod，
```

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 4]
```diff
@@ -1224,20 +1224,35 @@ See the [resourceQuota design doc](https://git.k8s.io/community/contributors/des
...
+此准入控制器会根据相应容器运行时类中定义的值在Pod中设置 `.spec.overhead`。
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:1250]:
```suggestion
此准入控制器会根据相应 RuntimeClass 中定义的值为 Pod 设置 `.spec.overhead`。
```

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 5]
```diff
@@ -1224,20 +1224,35 @@ See the [resourceQuota design doc](https://git.k8s.io/community/contributors/des
...
+
+{{< note >}}
+Pod 的 `.spec.overhead` 字段和容器运行时类的 `.overhead` 字段均为处于 beta 版本。
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:1253]:
```suggestion
Pod 的 `.spec.overhead` 字段和 RuntimeClass 的 `.overhead` 字段均为处于 beta 版本。
```

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 6]
```diff
@@ -1224,20 +1224,35 @@ See the [resourceQuota design doc](https://git.k8s.io/community/contributors/des
...
+如果你未启用 `PodOverhead` 特性门控，则将所有 Pod 视为未设置 `.spec.overhead`。
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:1254]:
```suggestion
如果你未启用 `PodOverhead` 特性门控，则所有 Pod 均被视为未设置 `.spec.overhead`。
```

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 7]
```diff
@@ -1224,20 +1224,35 @@ See the [resourceQuota design doc](https://git.k8s.io/community/contributors/des
...
++{{< feature-state for_k8s_version="v1.20" state="stable" >}}
 
-[RuntimeClass](/zh/docs/concepts/containers/runtime-class/) 定义描述了与运行 Pod
-相关的开销。此准入控制器将相应地设置 `pod.spec.overhead` 字段。
+如果你开启 `PodOverhead`
+[特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/),
+并且通过[Pod 开销](/zh/docs/concepts/scheduling-eviction/pod-overhead/)
+配置来定义一个 RuntimeClass， 这个准入控制器会检查新的 Pod。
+当启用的时候，这个准入控制器会拒绝任何 overhead 字段已经设置的 Pod。
+对于配置了 RuntimeClass 并在其 `.spec` 中选定 RuntimeClass 的Pod，
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:1249]:
```suggestion
对于配置了 RuntimeClass 并在其 `.spec` 中选定 RuntimeClass 的 Pod，
```

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 8]
```diff
@@ -1224,20 +1224,35 @@ See the [resourceQuota design doc](https://git.k8s.io/community/contributors/des
...
+### 容器运行时类 {#runtimeclass}
 
-{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
++{{< feature-state for_k8s_version="v1.20" state="stable" >}}
 
-[RuntimeClass](/zh/docs/concepts/containers/runtime-class/) 定义描述了与运行 Pod
-相关的开销。此准入控制器将相应地设置 `pod.spec.overhead` 字段。
+如果你开启 `PodOverhead`
+[特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/),
+并且通过[Pod 开销](/zh/docs/concepts/scheduling-eviction/pod-overhead/)
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:1246]:
```suggestion
并且通过 [Pod 开销](/zh/docs/concepts/scheduling-eviction/pod-overhead/)
```

Comment (tengqm) [reply content/zh/docs/reference/access-authn-authz/admission-controllers.md]: ?

**Comment by tengqm** [comment]
change looks good to me.
pls squash your 11 commits into one so we can merge it.

**Comment by sniperking1234** [comment]
@tengqm Done.

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 9]
```diff
@@ -1224,20 +1224,35 @@ See the [resourceQuota design doc](https://git.k8s.io/community/contributors/des
...
+
+{{< note >}}
+The `.spec.overhead` field for Pod and the `.overhead` field for RuntimeClass are both in beta. If you do not enable the `PodOverhead` feature gate, all Pods are treated as if `.spec.overhead` is unset.
+{{< /note >}}
 
 See also [Pod Overhead](/docs/concepts/scheduling-eviction/pod-overhead/)
 for more information.
 -->
-### 容器运行时类 {#runtimeclass} 
+### 容器运行时类 {#runtimeclass}
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:1240]: 将1240改为：

```
### RuntimeClass
```

这一子标题不必翻译。

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md (hunk 2 above)
Comment (sniperking1234) [reply content/zh/docs/reference/access-authn-authz/admission-controllers.md]: The title has been revised.

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md (hunk 9 above)
Comment (sniperking1234) [reply content/zh/docs/reference/access-authn-authz/admission-controllers.md]: The title has been revised.

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**